### DISCUSSION TITLE
curl error 23 while downloading https://repo.backpackforlaravel.com/packages.json:

### DISCUSSION THREAD

**Discussion by blondie63** [Laravel-Backpack/community-forum#911] [Q&A (Help)] [answered]
Today i've a very strange issue:
<img width="1067" alt="BKP6-composer-outdated" src="https://github.com/Laravel-Backpack/community-forum/assets/1261851/af28c36b-e663-4efc-a0f6-61aa69cbc16e">

The error message is ambiguous because it says "error downloading" but then refers to a writing error!
So I did all the checks on my Mac (cache, php settings, etc etc) but everything is ok and until last Saturday I didn't have any problems..
Then I saw that line 329 dir CurlDownloader.php does not exist!
CurlDownloader.php only has 84 lines!

Then I tried to open another project of mine that doesn't use BackPack and the "composer outdated" works correctly!

In the end I tried to open the link https://repo.backpackforlaravel.com/packages.json from the browser and it opened without problems!

At this point I don't understand if the problem is with the BackPack servers or something else, does anyone have an idea?

**Comment by karandatwani92** [comment]
Hey @blondie63 

I'm unable to understand what you are trying to do. The screenshot is different than your explanation.
Please provide some more info with a related screenshot. If it is already solved, remember to close the issue. Thanks

**Reply by blondie63** [reply]
Sorry, maybe my english is not correct..
As you can see in screenshot I simply run this command in my laravel backpack projects: composer ourdated

If i run same command in other projects on same Mac i've no errors !

This thing has been happening for a few days, no problems before

Was I understandable enough?

**Reply by blondie63** [reply]
Hi @tabacitu, is there some problem with my License ?
Username: mauromiotello120134800
All cmds composer update & outdated accessing your repo fails :( 

ps. take a look here: https://repo.backpackforlaravel.com/
there are some old links (404) like https://github.com/backpack/notification-manager

**Reply by tabacitu** [reply]
Hi Mauro,

From the point of view of access everything looks fine, you can check for yourself in your Backpack account - https://backpackforlaravel.com/user/tokens

The repo looks ok too, nothing wrong there. Notification Manager isn't an old link, it's an active package, that is hidden. We haven't polished it enough for people to use. But it's hosted on our repo as well, for when we do.

No backpack-related problems here.

**Comment by tabacitu** [comment]
Hey Mauro,

I just ran `composer outdate` and `composer update` on a few projects that use Backpack - they both work just fine!

Sounds like it's something specific to your computer or your projects. Think of what has changed in your project/system since it last worked... and take it from there. I don't think there's anything we can do to help.

Cheers!

**Comment by blondie63** [comment, marked as answer]
Hi @tabacitu Maybe it can be useful to someone, I solved it like this:
brew reinstall curl
ref: https://github.com/composer/composer/issues/11913

**Reply by pxpm** [reply]
It will for sure by the amount of reports on the mentioned issue. 

Thanks for sharing back the solution @blondie63 🙏
